Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "northstar-discord-rpc"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+rrplug = "0.1.0"
+discord-presence = { git = "https://github.com/jewlexx/discord-presence" } # TODO: move this to a version when the activity blocking is fixed
```

Comment:
Do you know if we have an ETA for this. Cause we might wanna wait for it to be tagged and uploaded to crates.io

Alternatively, make sure to at least specify commit so that it doesn't break over time cause author has pushed breaking changes that are to be addressed in this codebase.